packages/augur-ui/src/modules/common/buttons.styles.less

```diff
@@ -1006,6 +1006,70 @@
   }
 }
 
+.BettingBackLayButton {
+  .Button;
+
+  background: transparent;
+  color: var(--color-primary-text);
+  padding: @size-6 @size-12;
+
+  > div:first-of-type {
+    .text-12-bold;
+
+    color: var(--color-primary-text);
+    text-align: center;
+  }
+
+  > div:last-of-type {
+    .text-12;
+
+    color: var(--color-primary-text);
+    text-align: center;
+  }
+
+  &.Back {
+    border: @size-1 solid var(--color-outcome-five);
+
+    &:disabled {
+      background: var(--color-inactive-text);
```

Comment:
this should be `background: var(--color-positive-15);` with no opacity defined.

Reply:
Updated. Thank you.